### PR TITLE
Improved mobile views (Agenda list, motion block list)

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: client/src/app/site/motions/modules/motion-block/components/motion-block-detail/motion-block-detail.component.html [hunk 1]
```diff
@@ -39,9 +39,15 @@ <h2 *ngIf="block">
     (dataSourceChange)="onDataSourceChange($event)"
 >
     <!-- Title column -->
-    <div *pblNgridCellDef="'title'; row as motion; rowContext as rowContext" class="cell-slot fill motion-block-title">
+    <div *pblNgridCellDef="'title'; row as motion; rowContext as rowContext" class="cell-slot fill motion-block-title one-line">
         <a class="detail-link" [routerLink]="motion.getDetailStateURL()" *ngIf="!isMultiSelect"></a>
-        <span>{{ motion.getTitle() }}</span>
+        <div class="ellipsis-overflow">
+            <span *ngIf="motion.identifier">
```
Comment (tsiegleauq) [inline client/src/app/site/motions/modules/motion-block/components/motion-block-detail/motion-block-detail.component.html]: Should probably be dependent on the option: `Show the sequential number for a motion`

File: client/src/styles.scss [hunk 2]
```diff
@@ -816,6 +820,10 @@ button.mat-menu-item.selected {
     font-size: 16px;
 }
 
+.overflow-hidden {
```
Comment (tsiegleauq) [inline client/src/styles.scss]: That **could** cause whole vscroll lists or columns to flicker-to-disappear

File: client/src/app/site/motions/modules/motion-block/components/motion-block-detail/motion-block-detail.component.html (hunk 1 above)
Comment (emanuelschuetze) [reply client/src/app/site/motions/modules/motion-block/components/motion-block-detail/motion-block-detail.component.html]: I added the same multi-line layout as in motion-list - with submitters, sqeuential number etc.

File: client/src/styles.scss (hunk 2 above)
Comment (emanuelschuetze) [reply client/src/styles.scss]: I removed overflow-hidden class and used the same classed as in agenda list.

File: client/src/app/site/motions/modules/motion-block/components/motion-block-detail/motion-block-detail.component.scss [hunk 3]
```diff
@@ -3,3 +3,28 @@
 .edit-form {
     overflow: hidden;
 }
+
+
+.title-line {
+    font-weight: 500;
+    font-size: 16px;
+
+    .attached-files {
```
Comment (tsiegleauq) [inline client/src/app/site/motions/modules/motion-block/components/motion-block-detail/motion-block-detail.component.scss]: don't just copy css code. We don't want to adjust 2 files for the same change.

File: client/src/app/site/motions/modules/motion-block/components/motion-block-detail/motion-block-detail.component.ts [hunk 4]
```diff
@@ -142,6 +144,10 @@ export class MotionBlockDetailComponent extends BaseListViewComponent<ViewMotion
                 }
             })
         );
+        // load config variables
+        this.configService
+        .get<boolean>('motions_show_sequential_numbers')
```
Comment (tsiegleauq) [inline client/src/app/site/motions/modules/motion-block/components/motion-block-detail/motion-block-detail.component.ts]: linting will fail here

**Pull request merged**